docs/dev/test-configuration-options.md

```diff
@@ -597,6 +597,12 @@ Enables the interception of biometric input, allowing the test to simulate Touch
 
 Enables the use of the app's private app container directory instead of the shared app group container directory. For testing on the Real Device Cloud, the app gets resigned, which is why the shared directory is not accessible.
 
+---
+### `networkCapture`
+<p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span></small> | </p>
```

Comment:
Hi @kristofmuhi ! I made a few changes here -- is this correct? feel free to apply more edits! I removed "Android and iOS" from the header, since it is assumed to be both unless otherwise specified.

@aoleszyn